whistl/0.create_compendium_subset.py

```diff
@@ -0,0 +1,41 @@
+'''Save the subset of hte compendium that has already been labeled to allow faster loading'''
+import argparse
+
+import pandas as pd
+
+import utils
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser('This script saves the subset of the compendium that has '
+                                     'already been labeled to allow faster loading and training')
+    parser.add_argument('compendium', help='The refine.bio file containing the gene expression '
+                                            'data to subset')
+    parser.add_argument('map_file', help='The file produced by label_samples with sample labels.')
+    parser.add_argument('out_path', help='The location to write the resulting tsv to')
+    args = parser.parse_args()
+
+    # Next ~20 lines graciously lifted from comendium_eda.ipynb
```

Comment:
Typo (comendium -> compendium)

Also might be good to have the full filepath here, just so it's clear where this comes from (although I guess that may change, up to you)

Reply:
Wow typing is hard

Good call on the file path, went ahead and gave a full path